packages/models-library/src/models_library/service_settings_labels.py

```diff
@@ -137,6 +143,16 @@ class DynamicSidecarServiceLabels(BaseModel):
         ),
     )
 
+    restart_policy: RestartPolicy = Field(
+        RestartPolicy.NO_RESTART,
```

Comment:
if the dynamic sidecar has a compose (i.e. multiple containers). Does this restart applies to all of them or only the one connected to the reverse proxy?

Reply:
When it restarts, it restarts all the containers.